### ISSUE TITLE
Size of repo is too large

### ISSUE THREAD

**Issue by Sunnycheey** [allenai/scibert#66]
could you make the dataset(including training set, test set...) stored in other place?

**Comment by ibeltagy** [comment]
maybe something like this will make it faster to clone  https://stackoverflow.com/questions/600079/how-do-i-clone-a-subdirectory-only-of-a-git-repository/52269934#52269934

**Comment by Sunnycheey** [comment]
It's a solution, but is that ok with large resource added in source code?
